pages/docs/fundamentals/fundamentals-overview.md

```diff
@@ -7,6 +7,31 @@ redirect_from:
   - /features/
 ---
 
+## Call for contributions!
+
+**The call for contributions to the **2024 preCICE workshop** is now open! Use this form to submit your abstract for a 20-minute talk or a poster.**
+
+The 4th **preCICE workshop** will take place on **September 24-27**, on presence, at the **University of Stuttgart, Germany**.
+
+The workshop will consist of two parts: 
+- In the first part of the workshop (24-25 September), there will be a **course** on preCICE. The course is suited for both beginners and non-beginners, since advanced topics will be covered.
+- In the second part of the workshop (25-27 September), we will alternate **talks** with more interactive activities, such us the world cafe and user support sessions. There will also be a **poster session**.
+
+
+You can now submit your abstract for a talk or a poster using the form
+
+Registration for the workshop will open in mid April. Participants will be able to select between a **course + talks ticket** and **talks-only ticket**. 
+Table of approximate prices for registrations until the end of June:
+
+| Ticket type   | Members of academia | Members of industry  |
+|--------------|-----------|------------|
+| **course + talks** | 300 Euro    | 500  Euro     |
+| **talks-only**      | 200 Euro  | 300 Euro       |
```

Comment:
Note that our workshop is much more than talks (world café, support sessions, ...), and I think it would be good to advertise it as such. Let's call it "main workshop" instead of "talks only".